TypeApprovalRegister.yaml

```diff
@@ -1738,6437 +1534,490 @@ paths:
-                              - uuid: 'tar-1-0-0-application-p-0000'
-                                profile-name: 'application-profile-1-0:PROFILE_NAME_TYPE_APPLICATION_PROFILE'
-                                application-profile-1-0:application-profile-pac:
-                                  application-profile-capability:
-                                    application-name: 'RegistryOffice'
-                                    release-number: '1.0.0'
-                                  application-profile-configuration:
-                                    approval-status: 'application-profile-1-0:APPROVAL_STATUS_TYPE_APPROVED'
-                      logical-termination-point:
-                        type: array
-                        uniqueItems: true
-                        items:
-                          type: object
-                          required:
-                            - uuid
-                            - ltp-direction
-                            - client-ltp
-                            - server-ltp
-                            - layer-protocol
-                          properties:
-                            uuid:
-                              type: string
-                            ltp-direction:
-                              type: string
-                              enum:
-                                - 'core-model-1-4:TERMINATION_DIRECTION_BIDIRECTIONAL'
-                                - 'core-model-1-4:TERMINATION_DIRECTION_SINK'
-                                - 'core-model-1-4:TERMINATION_DIRECTION_SOURCE'
-                                - 'core-model-1-4:TERMINATION_DIRECTION_UNDEFINED_OR_UNKNOWN'
-                            client-ltp:
-                              type: array
-                              uniqueItems: true
-                              items:
-                                type: string
-                            server-ltp:
-                              type: array
-                              uniqueItems: true
-                              items:
-                                type: string
-                            layer-protocol:
-                              type: array
-                              minItems: 1
-                              maxItems: 1
-                              items:
-                                oneOf:
-                                  - description: 'operation server'
-                                    type: object
-                                    required:
-                                      - local-id
-                                      - layer-protocol-name
-                                      - operation-server-interface-1-0:operation-server-interface-pac
-                                    properties:
-                                      local-id:
-                                        type: string
-                                      layer-pr
```

Comment:
In "components :: schemas :: genericRepresentation", 
- [ ] response-value-list attributes description to be updated with "uuid=tar-2-0-0-response-p-\*" instead of \*-1-0-0-response-p-\* 
- [ ] consequent-action-list attributes description to be updated with "uuid=tar-2-0-0-action-p-\*" instead of \*-1-0-0-action-p-\*

Reply:
Changed as proposed.